waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/DFExecutorPriorityStrategies.scala

```diff
@@ -0,0 +1,99 @@
+package com.coxautodata.waimak.dataflow
+
+/**
+  * Defines various priority strategies for [[ DataFlowExecutor ]] to use.
+  *
+  * Created by Alexei Perelighin on 24/08/2018.
+  */
+object DFExecutorPriorityStrategies {
+
+  type actionQueue[C] = Seq[DataFlowAction[C]]
+
+  type priorityStrategy[C] = actionQueue[C] => actionQueue[C]
+
+  /**
+    * Default strategy, at the moment it will not do anything.
+    *
+    * @tparam C
+    * @return
+    */
+  def defaultPriorityStrategy[C]: PartialFunction[actionQueue[C], actionQueue[C]] = doNothing[C]
```

Comment:
I'm wondering whether the default strategy should be the fast track one as that's what I would want to use by default. Thoughts?

Reply:
done